package.nls.json

```
@@ -150,6 +150,7 @@
     "InterpreterQuickPickList.enterPath.placeholder": "Enter path to a Python Interpreter.",
     "InterpreterQuickPickList.browsePath.label": "Browse...",
     "InterpreterQuickPickList.browsePath.detail": "Browse your file system to select a Python Interpreter.",
+    "diagnostics.upgradeCodeRunner": "Please update Code Runner extension for it to be compatible with the Python extension.",
```

```suggestion
    "diagnostics.upgradeCodeRunner": "Please update the Code Runner extension for it to be compatible with the Python extension.",
```
Recommendation by /cc @luabud